### PR TITLE
recommiting mesheryctl system missing pages

### LINKED ISSUE #2935: [Docs] `mesheryctl system reset` page missing
#### Current State
No page exists for this command in the [mesheryctl Command Reference](https://docs.meshery.io/reference/mesheryctl).

#### Desired State
- A subcommand page should be dedicated to `mesheryctl system reset`.

#### Implementation
1. Create an entry for this subcommand in https://github.com/layer5io/meshery/tree/master/docs/_data/mesheryctlcommands/

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### LINKED ISSUE #2937: [Docs] `mesheryctl system logs` page missing
#### Current State
No page exists for this command in the [mesheryctl Command Reference](https://docs.meshery.io/reference/mesheryctl).

#### Desired State
A docs page for this subcommand should be available.

#### Implementation
1. Create an entry for this subcommand in https://github.com/layer5io/meshery/tree/master/docs/_data/mesheryctlcommands/

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### LINKED ISSUE #2943: [Docs] `mesheryctl system restart` page missing
#### Current State
No page exists for this command in the [mesheryctl Command Reference](https://docs.meshery.io/reference/mesheryctl).

#### Desired State
A docs page for this subcommand should be available.

#### Implementation
1. Create an entry for this subcommand in https://github.com/layer5io/meshery/tree/master/docs/_data/mesheryctlcommands/

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Created subcommands for mesheryctl system logs, restart and reset.

This PR, 
- fixes #2937 
- fixes #2935 
- fixes #2943 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by alphaX86 — COMMENTED** [review]
image
 We can either add one standard flag --help or remove this part

image

Plus, the logs page can be linked

File: docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-logs.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+<pre class="codeblock-pre">
+  <div class="codeblock">
+    {% for flag_hash in site.data.mesheryctlcommands.lifecycle.logs.flags %}{% assign flag = flag_hash[1] %}
+    {{ flag.flag }} # {{ flag.description }}
+    {% endfor %}
+  </div>
+</pre>
+<br/>
```
Comment (alphaX86) [inline docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-logs.md:65]:
```suggestion

```
Either removed or `--help` can be added

**Review by alphaX86 — COMMENTED** [review]
image

Repetition found! 😶

**Comment by alphaX86** [comment]
Hmm, one sec... I'll check if I can push those changes

**Comment by Anita-ihuman** [comment]
Alright then

**Comment by alphaX86** [comment]
@Anita-ihuman seems I can't co-author commits as it's limited to maintainers only..... Hmm, in that case I need to say few things IG

1. Some YAML files are duplicated, so you need to delete these files! 
     - `./_data/mesheryctlcommands/lifecycle/subcommands/log.yaml`
     - `./_data/mesheryctlcommands/lifecycle/subcommands/restart.yaml`
     
2. Change the rowspan to 18 at the mesheryctl-reference.md

File: docs/pages/reference/mesheryctl-commands.md [hunk 2]
```diff
@@ -138,6 +138,45 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
...
+      <td></td>
+      <td>{{ subcommand.description }}</td>
+    </tr>
+    {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system.log.flag %}{% assign flag = flag_hash[1] %}
+      <tr>
+        <td>{{ flag.flag }}</td>
+        <td>{{ flag.description }}</td>
+      </tr>
+    {% endfor %}
+  {% endfor %}
```
Comment (alphaX86) [inline docs/pages/reference/mesheryctl-commands.md:166]:
```suggestion
  {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.logs.command %}{% assign subcommand = subcommand_hash[1] %}
    <tr>
      <td><a href="{{ site.baseurl }}/reference/mesheryctl/system/logs">{{ subcommand.name }}</a></td>
      <td></td>
      <td>{{ subcommand.description }}</td>
    </tr>
    {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system.logs.flag %}{% assign flag = flag_hash[1] %}
      <tr>
        <td>{{ flag.flag }}</td>
        <td>{{ flag.description }}</td>
      </tr>
    {% endfor %}
  {% endfor %}
```

**Review by alphaX86 — CHANGES_REQUESTED** [review]
Along with those 2 changes, do this also

**Comment by alphaX86** [comment]
Then we're good to go

**Comment by leecalcote** [comment]
Merge conflict - https://github.com/layer5io/meshery/pull/3142/conflicts

**Review by leecalcote — APPROVED** [review]
Thank you, MeshMate @Anita-ihuman 💪

**Comment by Anita-ihuman** [comment]
Thank you everyone, for the tips, learnt a lot.

**Comment by ramrodo** [comment]
@leecalcote Sorry if I am insistent. This PR should not be merged until we fix those failing Netlify tests. I know that you or anyone else don't want to be a blocker for those contributors that help a lot but, if we don't solve these bad practices it will be harder to be accepted as a good project for the CNCF or other Foundation. We need to have best practices to have a project reliable.

One thing that we can do in the meantime is to **not allow** maintainers to merge PRs if the tests fail. We can configure that in _"Settings / Branches / Branch protection rules / master / **Include administrators**"._

If there is a PR where the tests are falining please ping me, I'll reply ASAP. I think there is no a huge problem to wait some hours or even a couple of days to fix the tests first.

**Pull request merged**